src/main/java/jenkins/plugins/clangscanbuild/ClangScanBuildDescriptor.java

```diff
@@ -105,8 +104,8 @@ public boolean configure( StaplerRequest req, JSONObject formData ) throws FormE
         return super.configure( req, formData );
     }
 
-    public boolean isApplicable( @SuppressWarnings("rawtypes") Class<? extends AbstractProject> jobType ){
-        return AbstractProject.class.isAssignableFrom( jobType );
+    @Override
+    public boolean isApplicable(Class<? extends AbstractProject> jobType) {
+    	return true;
```

Comment:
Adds consistency to ClangScanBuildDescriptor isApplicable and ClangScanBuildPublisherDescriptor isApplicable methods after remove the System.err. All instances that receives these methods are AbstractProject.